### PR TITLE
feat: supported tokenize and truncate_prompt endpoints

### LINKED ISSUE #25: Support tokenize and truncate_prompt endpoints

### PR DESCRIPTION
**Pull request by adubovik**
Resolves #25

### CODE REVIEW

**Comment by adubovik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-vertexai-pr-69.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by adubovik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-vertexai-pr-69.nightly-test.deltixhub.io
>E2E tests status: failed

File: aidial_adapter_vertexai/chat_completion.py [hunk 1]
```diff
@@ -86,3 +108,65 @@ async def generate_response(usage: TokenUsage, choice_idx: int) -> None:
...
+            tokens = await model.count_completion_tokens(value)
+            return TokenizeSuccess(token_count=tokens)
+        except Exception as e:
+            return TokenizeError(error=str(e))
+
+    async def tokenize_request(
+        self, model: ChatCompletionAdapter, request: ChatCompletionRequest
+    ) -> TokenizeOutput:
+        try:
+            prompt = await model.parse_prompt(request.messages)
```
Comment (Oleksii-Klimov) [inline aidial_adapter_vertexai/chat_completion.py:151]: May it return an error object instead of a prompt?

Comment (adubovik) [reply aidial_adapter_vertexai/chat_completion.py:151]: Indeed. Fixed.

**Comment by adubovik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-vertexai-pr-69.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Oleksii-Klimov — APPROVED** [review]

**Pull request merged**